src/main/kotlin/no/nav/dagpenger/oppslag/arena/ArenaClientSoap.kt

```diff
@@ -1,24 +1,27 @@
 package no.nav.dagpenger.oppslag.arena
 
+import no.nav.arena.services.lib.sakvedtak.SaksInfo
+import no.nav.arena.services.lib.sakvedtak.SaksInfoListe
+import no.nav.arena.services.sakvedtakservice.Bruker
+import no.nav.arena.services.sakvedtakservice.SakVedtakPortType
 import no.nav.tjeneste.virksomhet.behandlearbeidogaktivitetoppgave.v1.binding.BehandleArbeidOgAktivitetOppgaveV1
-import no.nav.tjeneste.virksomhet.behandlearbeidogaktivitetoppgave.v1.informasjon.Oppgave
-import no.nav.tjeneste.virksomhet.behandlearbeidogaktivitetoppgave.v1.informasjon.Oppgavetype
-import no.nav.tjeneste.virksomhet.behandlearbeidogaktivitetoppgave.v1.informasjon.Person
-import no.nav.tjeneste.virksomhet.behandlearbeidogaktivitetoppgave.v1.informasjon.Tema
+import no.nav.tjeneste.virksomhet.behandlearbeidogaktivitetoppgave.v1.informasjon.*
```

Comment:
Spotless does not like wildcard imports, it's best to just turn it off in intellij: https://www.jetbrains.com/help/idea/creating-and-optimizing-imports.html

Reply:
we should use klint rules 
download ktlint 

```
brew install shyiko/ktlint/ktlint

ktlint --apply-to-idea-project

```

Hit ` Option ` + `Command ` + ` L`   and `Option` + `Command` + `o` in intellij